test/runtime/kafka.go

```diff
@@ -34,13 +34,10 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 
```

Comment:
Typo in commit: "This was an incorrect way to check if the kafka broker if up or not" --> "This was an incorrect way to check if the kafka broker is up or not

Reply:
"waits till" --> "waits until"
"times outs" --> "times out".

Reply:
done